packages/runtime/container-runtime/src/channelCollection.ts

```diff
@@ -881,6 +881,7 @@ export class ChannelCollection implements IFluidDataStoreChannel, IDisposable {
 	public async getDataStore(
 		id: string,
 		requestHeaderData: RuntimeHeaderData,
+		requestedNodePath: string,
```

Comment:
I would prefer passing in the entire `request` here and logging the `request.url` and maybe the headers in the request too (which may be different from `requestHeaderData`). Basically, logging the original request url with the query params and everything might be better for debuggability.